src/index.js

```diff
@@ -1,13 +1,107 @@
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
-  return element;
-}
+const tasklist = new TaskList();
+const todoList = [...tasklist.getTasks()];
+const element = document.getElementById('tasks');
 
-document.body.appendChild(component());
\ No newline at end of file
+todoList.forEach((task) => {
+  const todo = `
+  <li class="list-items">
+    <span>
+      <input id=${task.index} class="checkbox-inputs" type="checkbox" value=${task.completed} /> ${task.description}
+    </span>
+    <span id=${task.index} class="task-action">
+      <i data-visible="true" id="menu-${task.index}" class="bi bi-three-dots-vertical ${task.index}"></i>
+      <i data-visible="false" id="trash-${task.index}" class="bi bi-trash ${task.index}"></i>
+    </span> 
+  </li>`;
+
+  element.insertAdjacentHTML('beforeend', todo);
+
+  const removeBtn = document.getElementById(`trash-${task.index}`);
+  removeBtn.addEventListener('click', () => {
+    const tasklist = new TaskList();
+    tasklist.removeTask(parseInt(removeBtn.id.substring(6), 10));
+  });
+});
+
+element.innerHTML += '<li class="last-item"><button id="btn-delete" class="btn-clear">Clear all completed</button></li>';
+
+const spans = document.querySelectorAll('.task-action');
+[...spans].forEach((span) => {
+  span.addEventListener('click', () => {
+    const id = span.getAttribute('id');
+    const actionBtns = document.getElementsByClassName(id);
+    [...actionBtns].forEach((btn) => {
+      const visibility = btn.getAttribute('data-visible');
+      if (btn.id.substring(0, 5) === 'trash') {
+        btn.addEventListener('click', () => {
+          const tasklist = new TaskList();
+          tasklist.removeTask(parseInt(btn.id.substring(6), 10));
+          location.reload();
+        });
+      }
+      if (visibility === 'true') {
+        btn.setAttribute('data-visible', false);
+      } else {
+        btn.setAttribute('data-visible', true);
+      }
+    });
+  });
+});
+
+const addBtn = document.getElementById('add-btn');
+addBtn.addEventListener('click', () => {
+  const description = document.forms.task.description.value;
+  const task = new Task(0, description, false);
+  const tasklist = new TaskList();
+  tasklist.addNewTask(task);
+  location.reload();
+});
+
+const checkBoxInputs = document.querySelectorAll('.checkbox-inputs');
+const deletelist = [];
+[...checkBoxInputs].forEach((btn) => {
+  btn.addEventListener('change', (event) => {
+    if (btn.checked) {
+      deletelist.push(parseInt(btn.id, 10));
+    } else {
+      deletelist.forEach((num, index) => {
+        if (parseInt(btn.id, 10) === num) {
+          deletelist.splice(index, 1);
+        }
+      });
+    }
+  });
+});
+
+const deleteBtn = document.getElementById('btn-delete');
+deleteBtn.addEventListener('click', () => {
+  const tasklist = new TaskList();
+  deletelist.forEach((id) => {
+    tasklist.removeTask(id);
+  });
+  location.reload();
```

Comment:
- [ ] Kindly add a new JavaScript file for the new functionality as per the requirement. :+1: